index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Theoneste Nsanzabarinda</title>
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-95YMP3C67Q"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+
+      gtag("config", "G-95YMP3C67Q");
+    </script>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="./CSS/style.css" />
+  </head>
+
+  <body>
+    <header class="main-header d-flex">
+      <h2>Welcome</h2>
+      <nav class="nav d-flex">
+        <ul class="nav-menu d-flex flex-center">
+          <li class="nav-link"><a href="#Portfolio">Portoflio</a></li>
+          <li class="nav-link active"><a href="#About">About</a></li>
+          <li class="nav-link"><a href="#Contact">Contact</a></li>
+        </ul>
+        <img src="./images/icons/Union.png" class="show-menu-btn" alt="union" />
+        <img src="./images/icons/Icon@2x.svg" class="closed-menu-btn" alt="" />
+      </nav>
+    </header>
+
+    <main class="container">
+      <section class="main-section">
+        <div class="main-section-title">
+          <h1 class="h2-size">Hey there. I’m Theo.</h1>
+          <h2 class="h2-size">I’m a software developer</h2>
+        </div>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="icons-container">
+          <ul class="social-media-icons d-flex flex-center">
+            <li>
+              <a href="#">
+                <img src="./images/icons/twitter.png" alt="twitter" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/icons/Linkedin-icon.png" alt="linkedin" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/icons/medium.png" alt="medium" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/icons/github.png" alt="github" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/icons/angellist.png" alt="angellist" />
+              </a>
+            </li>
+          </ul>
```

Comment:
[OPTIONAL]

Don't forget to add the links to your social media, it's the best way for people to reach you! :+1: